README.md

```diff
@@ -191,12 +191,13 @@ To achieve this, Bonsai is randomly toggling a digital output in the HARP behavi
 |                           |         Audio         |   213    |       -        |    Not in use Timestamped(FrameNumber)        |
 |                           |         Key           |   214    |       -        |    Not in use Timestamped(FrameNumber)        |
 |       **UBX**             |         UBX           |          |                |                                               |
-|       **Omnicept**        |      EyeTracking      |   215    |                |            Timestamped(long[])                |
-|                           |       HeartRate       |   216    |                |            Timestamped(long[])                |
-|                           |          IMU          |   217    |                |            Timestamped(long[])                |
-|                           |         Mouth         |   218    |                |            Timestamped(long[])                |
-|     **VRTransform**       |       VrTimestamp     |   219    |                |            Timestamped(long)                  |
-|     **UnityImage**        |       VrTimestamp     |   220    |                |            Timestamped(long)                  |
+|       **Omnicept**        |      EyeTracking      |   190    |                |            Timestamped(long[])                |
+|                           |       HeartRate       |   191    |                |            Timestamped(long[])                |
+|                           |          IMU          |   192    |                |            Timestamped(long[])                |
+|                           |         Mouth         |   193    |                |            Timestamped(long[])                |
+|     **VRTransform**       |       VrTimestamp     |   180    |                |            Timestamped(long)                  |
+|    **GeoReference**       |       VrTimestamp     |   181    |                |            Timestamped(long)                  |
```

Comment:
```suggestion
|    **Georeference**       |       VrTimestamp     |   181    |                |            Timestamped(long)                  |
```

Check if all other instances of `Georeference` throughout are all lower-case for consistency.